luigi/notifications.py

```diff
@@ -124,12 +126,10 @@ def send_email(subject, message, sender, recipients, image_png=None):
                  "-------------", recipients, sender, subject, message)
     if not recipients or recipients == (None,):
         return
-    if sys.stdout.isatty() or DEBUG:
+    if (sys.stdout.isatty() or DEBUG) and (not config.getboolean('email', 'force-send', False)):
```

Comment:
yeah i'm planning to change all config to use the new `Config` class anyway so I'll do it later